gateway/gateway_test.go

```diff
@@ -265,11 +338,41 @@ func TestGetDestChannelAdvanced(t *testing.T) {
 					hits[gw.Name]++
 					switch br.Account {
 					case "irc.zzz":
-						assert.Equal(t, []config.ChannelInfo{{Name: "#main", Account: "irc.zzz", Direction: "out", ID: "#mainirc.zzz", SameChannel: map[string]bool{"announcements": false}, Options: config.ChannelOptions{Key: ""}}, {Name: "#main-help", Account: "irc.zzz", Direction: "out", ID: "#main-helpirc.zzz", SameChannel: map[string]bool{"announcements": false}, Options: config.ChannelOptions{Key: ""}}}, channels)
```

Comment:
This change was required as test could fail because of wrong ordering of slice elements.